gcd/src/main/java/org/spine3/server/storage/datastore/tenant/DatastoreTenants.java

```diff
@@ -70,10 +76,13 @@ private DatastoreTenants() {
      *
      * <p>A single-tenant app (or a single-tenant BoundedContext in a multitenant app) does not
      * require a {@code TenantIndex} to be set explicitly, so this method assumes that it is an a
-     * single tenant context.
+     * single tenant context. Though, not using this method when creating an instance of
```

Comment:
Please revert this portion of changes. It's enough that we describe the details in `Contexts` and reference it above.